test/LowerColorado_TX/test_AnA_V4_NHD.yaml

```diff
@@ -124,10 +124,10 @@ output_parameters:
 #         stream_output_time: 1 #[hr]
 #         stream_output_type: '.nc' #please select only between netcdf '.nc' or '.csv' or '.pkl'
 #         stream_output_internal_frequency: 30 #[min] it should be order of 5 minutes. For instance if you want to output every hour put 60
-    parquet_output:
-        #---------
-        parquet_output_folder: output/
-        configuration: short_range
-        prefix_ids: nex
+#     parquet_output:
```

Comment:
Can you elaborate on why this was commented out?

Reply:
Just to simplify the yaml file. So if user need that, can uncomment it